client/lobby/SelectionTab.h

```diff
@@ -62,6 +65,24 @@ class SelectionTab : public CIntObject
 	std::shared_ptr<CAnimation> iconsVictoryCondition;
 	std::shared_ptr<CAnimation> iconsLossCondition;
 
+	class CMapInfoTooltipBox : public CWindowObject
+	{
+		const int IMAGE_SIZE = 169;
+		const int BORDER = 30;
+
+		bool drawPlayerElements;
+		bool renderImage;
+
+		std::shared_ptr<CFilledTexture> backgroundTexture;
+		std::shared_ptr<CTextBox> label;
+		std::shared_ptr<CPicture> image1;
+		std::shared_ptr<CPicture> image2;
+
+		Canvas createMinimap(std::unique_ptr<CMap> & map, int layer);
+		std::vector<std::shared_ptr<IImage>> redrawMinimap(ResourceID resource, int size=144);
```

Comment:
Also, `redrawMinimap` does not really redraws. Perhaps rename?
```suggestion
		Canvas createMinimapForLayer(std::unique_ptr<CMap> & map, int layer);
		std::vector<std::shared_ptr<IImage>> createMinimaps(ResourceID resource, int size=144);
```